### DISCUSSION TITLE
Vue 3公式ドキュメント翻訳

### DISCUSSION THREAD

**Discussion by kazupon** [vuejs-translations/docs-ja#273] [General]
# ようこそ、Vue.js 公式ドキュメント翻訳へ、そしてありがとうございます！

この discussion は、先日 `2/7` に公開された Vue 3 公式ドキュメントの翻訳をしたい人方向けの入り口となるサポートページです。
翻訳されたい方は、翻訳ガイドラインを一度一読し、以降にある翻訳対象まとめissuesからたどって翻訳をお願いします！

何か分からないことがありましたら、気軽にこの discussion にコメントください。

## Vue.js 公式サイト日本語翻訳ガイド
翻訳する上でかかせないガイドラインです。
https://github.com/vuejs-translations/docs-ja/blob/main/.github/CONTRIBUTING.md

## 翻訳対象まとめ issues
Vue.js 3 公式ドキュメントのカテゴライズした issue です。下記 issue リンク先には、Vue.js 公式ドキュメントのセクション単位 (1 markdown) でさらに issue リスト化されています。その中から翻訳したいところを選んで翻訳お願いします！

- https://github.com/vuejs-translations/docs-ja/issues/6
- https://github.com/vuejs-translations/docs-ja/issues/84
- https://github.com/vuejs-translations/docs-ja/issues/100
- https://github.com/vuejs-translations/docs-ja/issues/114

**Comment by naokie** [comment]
文末に出てくる「:」に疑問を持ったことはありませんか？
Vue.js のドキュメントでは、次の文（コードサンプルが多いです）にかかる意味で多用されているため、意味のある約物として原文を尊重しています。

**Comment by naokie** [comment]
TypeScript 対応のため増えている日本語・英語どちらの単語を使おうといった例を [用字、用語](https://github.com/vuejs-translations/docs-ja/wiki/%E7%94%A8%E5%AD%97%E3%80%81%E7%94%A8%E8%AA%9E) に増やしました。

**Comment by yamanoku** [comment]
> the latest standards

について「最新の標準」なのか「最新の規格」として訳そうか悩んでいます。

https://github.com/vuejs-translations/docs-ja/blob/06fd0cde9c5cb65da5b1055a4d2232076715dfca/src/guide/extras/web-components.md?plain=1#L251

**Reply by naokie** [reply]
ちょっと難しいですよね。
個人的に思うところありますので、参考までに。

単数形の「standard」に対応する「標準」という意味が考えのベースになると思います。
しかし、複数形になったときには一般的に「規格、基準」という訳が工業製品まわりに多い気がしています。
ところが Web 界隈では「Web Standards」を「ウェブ標準」という訳で出回ってしまっていて、この文で言うところも「標準化団体が定めている仕様の最新ウェブ標準」を指しているので、これに合わせるのがよいと思います。

参考: https://ja.wikipedia.org/wiki/%E3%82%A6%E3%82%A7%E3%83%96%E6%A8%99%E6%BA%96

**Reply by yamanoku** [reply]
ありがとうございます！
自分も「標準」が染み付いちゃってるのでそちらのが自然な方が読みやすいかな…

**Comment by jay-es** [comment]
前回、[コンポーザブル](https://github.com/vuejs-translations/docs-ja/blob/main/src/guide/reusability/composables.md)のページで、"composable function" → コンポーザブル関数としましたが、
[Reactivity Transform](https://github.com/vuejs-translations/docs-ja/blob/main/src/guide/extras/reactivity-transform.md) のページ翻訳で "composition function" というのが出てきて少し悩んでいます。

いきなり「合成関数」としても伝わらなそうですし「コンポジション関数」あたりでよいでしょうか？

**Reply by jay-es** [reply]
（"composable", "composable function" については用語集に追加しておきます）

**Reply by naokie** [reply]
むむむ、難しいですね

「合成関数」という用語は一般的に数学で使われていて、対応する英語は「composite [compound] function」のようです。
数学方面に疎いので詳細がわかりませんが、指すところの意味が違うなら、すでにある用語なので避けるべきものではあると思います。

また、Composition API / Options API は対比するものとして、英語をそのままにしています。
Composition API  が v3 における新しいものなので、英語のまま or カタカナ（英語併記）にして、馴染むまで置いとくのがよさそうですね。

**Comment by jay-es** [comment]
"ref" を「参照」と翻訳しているケースが多いですが、`ref()` の変数を指している場合は 「ref」のままがよいと思い、レビューでもそのように直してもらっています。
（例: ref のオブジェクト、など）

こちら認識問題なければ用語集に追加してもOKでしょうか？

**Reply by naokie** [reply]
まさにそのとおりですね 👍🏼

**Reply by kazupon** [reply]
自分もそれでいいと思います！
